lib/autocomplete-manager.coffee

```diff
@@ -190,8 +191,16 @@ class AutocompleteManager
     return unless suggestions.length
 
     unless @overlayDecoration?
-      marker = @editor.getLastCursor()?.getMarker()
-      @overlayDecoration = @editor?.decorateMarker(marker, { type: 'overlay', item: this })
+      if atom.config.get('autocomplete-plus.selectionsListFollows') == 'Cursor'
+        @currentWordMarker = @editor.getLastCursor()?.getMarker()
+      else
+        cursor = @editor.getLastCursor()
+        position = cursor?.getBeginningOfCurrentWordBufferPosition()
+        @currentWordMarker = @editor.markBufferPosition(position)
+
+      @compositeDisposable.add @currentWordMarker
```

Comment:
Not sure whether I understood the compositeDisposable completely. Is that what you meant by
> register it into the class compositeDisposable so it gets cleaned up
?